### PR TITLE
74: add entity_list create and add_property

### LINKED ISSUE #74: Add entity_list create and add_property
Will be supported when Central v2024.1 is released. 

https://github.com/getodk/central/issues/575

https://github.com/getodk/central/issues/576

### PR DESCRIPTION
**Pull request by lindsay-stevens**
Closes #74

#### What has been done to verify that this works as intended?

Added tests with example script.

#### Why is this the best possible solution? Were any other approaches considered?

Added `get` method as well because I think that would be useful and it was straightforward.

I spent a while on the example script trying to come up with a complete workflow of like: create form -> load submissions -> create entity list -> create entities etc., but I wasn't sure it was realistic or useful so I stuck with what Yaw suggested.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

New features!

#### Do we need any specific form for testing your changes? If so, please attach one.

Non

#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.

No just the auto-generated docs on release.

#### Before submitting this PR, please make sure you have:
- [x] included test cases for core behavior and edge cases in `tests`
- [x] run `python -m unittest` and verified all tests pass
- [x] run `ruff format pyodk tests` and `ruff check pyodk tests` to lint code
- [x] verified that any code or assets from external sources are properly credited in comments

### CODE REVIEW

File: docs/examples/create_entities_from_submissions/create_entities_from_submissions.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+"""
+A script that uses CSV data create an entity list and populate it with entities.
```
Comment (lognaturel) [inline docs/examples/create_entities_from_submissions/create_entities_from_submissions.py:2]: Missing "to"?

File: pyodk/_endpoints/entity_lists.py [hunk 2]
```diff
@@ -73,3 +120,73 @@ def list(self, project_id: int | None = None) -> list[EntityList]:
...
+        return EntityList(**data)
+
+    def create(
+        self,
+        approval_required: bool | None = False,
+        entity_list_name: str | None = None,
+        project_id: int | None = None,
+    ) -> EntityList:
+        """
+        Create an Entity.
```
Comment (lognaturel) [inline pyodk/_endpoints/entity_lists.py:163]: Entity -> Entity List

File: tests/endpoints/test_entity_lists.py [hunk 3]
```diff
@@ -19,7 +19,37 @@ def test_list__ok(self):
...
             with Client() as client:
                 observed = client.entity_lists.list()
-        self.assertEqual(2, len(observed))
+        self.assertEqual(3, len(observed))
         for i, o in enumerate(observed):
             with self.subTest(i):
                 self.assertIsInstance(o, EntityList)
+
+    def test_get__ok(self):
+        """Should an EntityList object."""
```
Comment (lognaturel) [inline tests/endpoints/test_entity_lists.py:28]: return

**Review by lognaturel — APPROVED** [review]
Couple of small comment issues but functionality and misc improvements look great, thank you!

File: tests/endpoints/test_entity_lists.py (hunk 3 above)
Comment (lindsay-stevens) [reply tests/endpoints/test_entity_lists.py:28]: sometimes it just should

Comment (lognaturel) [reply tests/endpoints/test_entity_lists.py]: No EntityList should ever obJECT.
